### DISCUSSION TITLE
Cardano ADA Ledger Live

### DISCUSSION THREAD

**Discussion by DarkIntegrity** [LedgerHQ/ledger-live-desktop#3742] [Q&A]
I know this has been brought up several times before. But I am brining this up again.

1. Cardaon ADA now has a market cap of 45 Billion. So it's top 10...
2. There is a lot of hype surrounding Cardano because smart contracts are coming soon, the Ethiopia deal, the erc20 converter, etc. So a lot of people are rightfully interested and investing.
3. Its fully supported on some completely free hot wallets. ex. Atomic Wallet (Allows sending, receiving and staking)
4. Supported by other hardware wallets, ex Ellipal Titan.

What's the issue exactly? What's excepted of IOHK? Can you guys be more transparent, if there are issues with IOHK can you please enumerate them so the community can take action?

**Comment by johnpc** [comment]
This may be a duplicate of https://github.com/LedgerHQ/ledger-live-desktop/discussions/4010
